### PR TITLE
Update tpacpi-bat

### PR DESCRIPTION
**Pull request by Ottaviocr**
Added override for Thinkpad Edge E130

### CODE REVIEW

**Comment by Ottaviocr** [comment]
See:
https://github.com/linrunner/TLP/issues/310

**Comment by linrunner** [comment]
OK imho.

File: tpacpi-bat [hunk 1]
```diff
@@ -394,6 +394,7 @@ sub acpiCall($){
   my %aslBaseOverrides = (
       'ThinkPad S2'         => '\_SB.PCI0.LPCB.EC.HKEY',
       'ThinkPad 13 2nd Gen' => '\_SB.PCI0.LPCB.EC.HKEY',
+	'ThinkPad Edge E130' => '\_SB.PCI0.LPCB.EC.HKEY',
```
Comment (ewolk) [inline tpacpi-bat]: ws

Comment (teleshoes) [reply tpacpi-bat]: whoops. i meant: whitespace

**Pull request merged**